README.md

```diff
@@ -10,7 +10,7 @@ F# source code formatter, inspired by [scalariform](https://github.com/mdr/scala
 
 ## Quick install
 
-* Command line: `dotnet tool install -g fantomas`
+* Command line: `dotnet tool install -g fantomas-tool`
```

Comment:
`fantomas-tool` is the name of the `v4` series.
Going forward (`v5`) uses `fantomas`.
I would accept a PR that adds `--prerelease` instead.
Please note that we are reviewing all documentation in https://github.com/fsprojects/fantomas/issues/2315. 
This readme will eventually be replaced altogether with the documentation website.